configs/components/openssl.rb

```diff
@@ -1,88 +1,215 @@
-component "openssl" do |pkg, settings, platform|
-  ## SOURCE METADATA
-  pkg.version "1.0.2m"
-  pkg.md5sum "10e9e37f492094b9ef296f68f24a7666"
-  pkg.url "https://www.openssl.org/source/openssl-#{pkg.get_version}.tar.gz"
+component 'openssl' do |pkg, settings, platform|
+  if settings[:runtime_project] == 'agent'
+    pkg.version '1.0.2k'
+    pkg.md5sum 'f965fc0bf01bf882b31314b61391ae65'
+    pkg.replaces 'pe-openssl'
+  else
+    pkg.version '1.0.2m'
+    pkg.md5sum '10e9e37f492094b9ef296f68f24a7666'
+  end
```

Comment:
I understand not wanting to introduce too many changes when moving to a runtime, but we should leave a comment in here to get rid of the block when agent is updated to latest openssl (there have been more security things). One of the nice things with the runtime is everything will get these updates together :)